Userland/Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp

```diff
@@ -522,7 +522,18 @@ CommandResult DisplayListPlayerSkia::push_stacking_context(PushStackingContext c
     }
 
     if (command.mask.has_value()) {
-        auto alpha_mask = alpha_mask_from_bitmap(*command.mask.value().mask_bitmap, command.mask.value().mask_kind);
+        auto mask_bitmap_or_error = Gfx::Bitmap::create(Gfx::BitmapFormat::BGRA8888, command.mask->mask_display_list->get_bitmap_rect().size().to_type<int>());
```

Comment:
Is it acceptable to render the `DisplayList` into a bitmap here? Or would it be better to try to merge the two `DisplayList`?

Reply:
We talked about this in Discord. For a first cut, let's always render to a separate bitmap first.